README.md

```diff
@@ -45,10 +45,9 @@ rm -rf package-lock.json
 
 ### USB connection
 First, make sure that your user is part of the `dialout` group if it is necessary in your Linux distribution. You can do this with the following command:
-```bash
-sudo usermod -a -G dialout youruser
 ```
-where **youruser** is the user name you are running the program with.
+sudo usermod -a -G dialout `whoami`
```

Comment:
looks great! perhaps having a link as reference for "Changing permissions on serial port" would be useful for new GNU/Linux user: https://askubuntu.com/questions/58119/changing-permissions-on-serial-port (there might be other references).

Reply:
@mxochicale - good point, I've added the link to the README.